### PR TITLE
welcome page in the addon

### PR DESCRIPTION
**Pull request by bakulf**

### CODE REVIEW

**Comment by jonathanKingston** [comment]
Can we use the non minified CSS too? It shouldn't add any size once the addon is zipped anyway and aids it being trimmed down here too.
We could probably do with a clean up task too to depend on a standard set of vars and parts from the doorhanger CSS too.

**Comment by jonathanKingston** [comment]
Also I think we need to make this translated right?

File: src/pages/welcome.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      <div class="flx flx-row h-padding v-padding header-bg">
+        <div class="logo"></div>
+        <div class="logotype"></div>
+      </div>
+    </header>
+    <main class="section document">
+      <div class="row">
+        <div class="document-content">
+          <h1 class="document-headline ff-Met">Welcome to the Firefox Private Network Beta</h1>
+          <p class="lead">Thanks for participating in this limited-time, free beta of Firefox Private Network. Firefox Private Network lets you use public Wi-Fi more safely and securely. It does this by encrypting the web addresses you visit and the data you send to most websites. This helps protect your web browsing from hackers, websites, and trackers.</p>
```
Comment (pdehaan) [inline src/pages/welcome.html]: Per https://github.com/mozilla/secure-proxy/issues/617#issue-492985348

Correct String | Incorrect String | Location
-- | -- | --
It does this by encrypting the web addresses you visit. | It does this by encrypting the web addresses you visit and the data you send to most websites. | 1st paragraph

File: src/pages/welcome.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+          <h2 class="ff-Met document-subhead">Your privacy</h2>
+          <p>Firefox cares about your privacy. Firefox Private Network doesn’t collect information about your browsing. We do collect generic information about how you use the extension, such as data about how you interact with it, when you turn it off, and technical information about how it functions. Our <a href="https://www.mozilla.org/privacy/firefox-private-network" target="_blank" rel="noopener noreferrer">privacy policy</a> describes how we handle this data. You may <a href="https://support.mozilla.org/kb/share-data-mozilla-help-improve-firefox?redirectslug=send-performance-data-improve-firefox" target="_blank" rel="noopener noreferrer">opt out</a> of sharing this data with us at anytime.</p>
```
Comment (pdehaan) [inline src/pages/welcome.html]: Per https://github.com/mozilla/secure-proxy/issues/617#issue-492985348

Correct String | Incorrect String | Location
-- | -- | --
You may opt out of sharing this data with us at any time. | You may opt out of sharing this data with us at anytime. | 2nd paragraph

Opt out link should no longer go to SUMO, but expand below the content. [Here is the UX for how this should look](https://designakt.github.io/SecureProxyBeta-PrivacyNotice/Firefox%20Private%20Network.html#instruction).

**Comment by bakulf** [comment]
I just updated the PR. Do we need Metropolis font? @lesleyjanenorton

**Comment by bakulf** [comment]
Translation can be done later. This is US only... right? I'll file an issue for the translation.

**Comment by pdehaan** [comment]
> Do we need Metropolis font? @lesleyjanenorton

Looks like we're using Metropolis in a few places in welcome.html (for the headline and subheads):

```sh
$ git grep -n ff-Met

src/pages/app.css:237:.ff-Met {
src/pages/welcome.html:22:          <h1 class="document-headline ff-Met">Welcome to the Firefox Private Network Beta</h1>
src/pages/welcome.html:25:          <h2 class="ff-Met document-subhead">Your privacy</h2>
src/pages/welcome.html:28:          <h2 class="ff-Met document-subhead">Using Private Network</h2>
src/pages/welcome.html:48:          <h2 class="ff-Met document-subhead">Giving feedback and getting updates</h2>
src/pages/welcome.html:61:          <h2 class="ff-Met document-subhead">Getting help and support</h2>
```

**Pull request merged**